src/PscIde/Command.purs

```diff
@@ -258,14 +264,16 @@ instance decodePursuitCompletion :: DecodeJson PursuitCompletion where
   decodeJson json = do
     o <- decodeJson json
     identifier <- o .? "ident"
-    type' <- o .? "type"
+    type' <- Right $ either (const Nothing) Just $ o .? "type"
```

Comment:
Looks like the DecodeJSON instance for Maybe already handles potential nulls:

https://github.com/purescript-contrib/purescript-argonaut-codecs/blob/026040796dd502a94c7a5b154356f2cc939b62a5/src/Data/Argonaut/Decode/Class.purs#L71-L74

So I think it should be fine to just `type' <- o .? "type"` here